AR-builder.groovy

```diff
@@ -308,208 +308,208 @@ pipeline {
+//                    }
+//                    steps {
+//                        script {
+//                            def trackName = env.WIRE_ANDROID_PROD_TRACK_NAME
+//                            echo 'Checking folder before prod playstore upload'
+//                            sh "ls -la app/build/outputs/bundle/${params.FLAVOR.toLowerCase()}${params.BUILD_TYPE.capitalize()}/"
+//                            echo "Uploading file to prod Playstore track ${trackName}"
+//                            androidApkUpload(
+//                                    googleCredentialsId: "${env.GOOGLE_PLAY_CREDS}",
+//                                    filesPattern: "app/build/outputs/bundle/${params.FLAVOR.toLowerCase()}${params.BUILD_TYPE.capitalize()}/com.wire.android-*.aab",
+//                                    trackName: trackName,
+//                                    rolloutPercentage: '100',
+//                                    releaseName: "${trackName} Release"
+//                            )
+//                        }
+//                    }
+//                }
+//
+//                stage('Upload to Wire Internal') {
+//                    when {
+//                        expression {
+//                            params.UPLOAD_TO_PLAYSTORE_ENABLED &&
+//                                    params.RUN_ACCEPTANCE_TESTS &&
+//                                    params.RUN_UNIT_TEST &&
+//                                    params.RUN_STATIC_CODE_ANALYSIS &&
+//                                    params.UPLOAD_TO_S3 &&
+//                                    params.FLAVOR == 'Internal' &&
+//                                    params.SOURCE_BRANCH == 'internal' &&
+//                                    params.BUILD_TYPE == 'Compat' &&
+//                                    params.CHANGE_ID == null
+//                        }
+//                    }
+//                    steps {
+//                        script {
+//                            def trackName = env.WIRE_ANDROID_INTERNAL_TRACK_NAME
+//                            echo 'Checking folder before prod playstore upload'
+//                            sh "ls -la app/build/outputs/bundle/${params.FLAVOR.toLowerCase()}${params.BUILD_TYPE.capitalize()}/"
+//                            echo 'Uploading file to prod Playstore track ${trackName}'
+//                            androidApkUpload(
+//                                    googleCredentialsId: "${env.GOOGLE_PLAY_CREDS}",
+//                                    filesPattern: "app/build/outputs/bundle/${params.FLAVOR.toLowerCase()}${params.BUILD_TYPE.capitalize()}/com.wire.android-*.aab",
+//                                    trackName: trackName,
+//                                    rolloutPercentage: '100',
+//                                    releaseName: "${trackName} Release"
+//                            )
+//                        }
+//                    }
+//                }
+//            }
+//        }
```

Comment:
Worth it to keep the comments? Might as well delete it. We can always use Git to see what was here before.

Reply:
some leftover